contributors/devel/instrumentation.md

```diff
@@ -38,13 +55,145 @@ first calling WithLabelValues if your metric has any labels
    https://github.com/kubernetes/kubernetes/blob/cd3299307d44665564e1a5c77d0daa0286603ff5/pkg/apiserver/apiserver.go#L87
```

Comment:
How about including short snippets of the code here instead of having link to source code?